### DISCUSSION TITLE
Potential fix for comms behavior problem.

### DISCUSSION THREAD

**Discussion by mrstealyommr** [ValveSoftware/Dota2-Gameplay#14905] [Ideas]
tldr: Have chat gpt like bot translate flaming in chat near real time to something constructive. 

With voice it would be trickier and much more costly. Probably a feature to shelve for 3-5 years from now.  Sell unaltered dota 2 chats to recoup some $. Could probably be used to build some ego maniac terminator llm.
 When people flame, they usually mean to criticize someone to better them, but when that person isn't receptive and still plays like someone who bought their account, it gets under peoples nerves. Thats when vitriol flies. Its a team game, theres always a weak link. Let flamers flame, just clean the speech up. let them see their own toxic chat and the other 9 players see slightly modified messages. Preventing someone from communicating completely is not the way, especially when there is a mute option every player can use by themselves.

**Comment by noxiouswire** [comment]
Your prose is interesting and instructive. [geometry dash meltdown](https://geometrydash-meltdown.co/)
